### PR TITLE
Update discovery protocol specs for v0.13

### PR DESCRIPTION
**Pull request by AudriusButkevicius**

### CODE REVIEW

File: specs/globaldisco-v3.rst [hunk 1]
```diff
@@ -7,17 +7,16 @@ Announcements
 -------------
 
 A device should announce itself at startup. It does this by an HTTPS POST to
-the announce server URL (with the path usually being "/", but this is of
-course up to the discovery server). The POST has a JSON payload listing direct
-connection addresses (if any) and relay addresses (if any)::
+the announce server URL (with the path usually being "/v2/", but this is of
+course up to the discovery server). The POST has a JSON payload listing
+connection addresses (if any)::
```
Comment (calmh) [inline specs/globaldisco-v3.rst]: Maybe note that the standard discovery server currently requires this path

File: specs/globaldisco-v3.rst [hunk 2]
```diff
@@ -7,17 +7,16 @@ Announcements
...
 	{
-		direct: ["tcp://192.0.2.45:22000", "tcp://:22202"],
-		relays: [{"url": "relay://192.0.2.99:22028", "latency": 142}]
+		addresses: ["tcp://192.0.2.45:22000", "tcp://:22202", "relay://192.0.2.99:22028"],
 	}
 
-It's OK for either of the "direct" or "relays" fields to be either the empty
-list (``[]``), ``null``, or missing entirely. An announcment with both fields missing
+It's OK for the "addresses" field to be either the empty list (``[]``),
+``null``, or missing entirely. An announcment with the field missing
```
Comment (calmh) [inline specs/globaldisco-v3.rst]: Typo on "announcement"

**Comment by calmh** [comment]
After that just merge

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as c4a160095087da6042cef223f13644c763cf190f. Thanks, @AudriusButkevicius!

**Pull request closed without merging**